portal/_static/custom.css

```
@@ -1,3 +1,27 @@
+main.banner-main #project-pythia {
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+}
+
+main.banner-main #project-pythia p {
+  font-size: 1.4rem;  /* default: 1.25rem */
+  /* font-weight: 700; default: 300 */
+}
+
+main.banner-main #project-pythia a,
+main.banner-main #project-pythia a:visited {
+  color: rgba(var(--spt-color-light), 1);
+  text-decoration: underline dotted rgba(var(--spt-color-gray-400), 1);
+}
+
+main.banner-main #project-pythia a.headerlink:hover {
+  color: #000;
```

That was dumb!  White, not black!

```suggestion
  color: #DDD;
```